en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md

```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
 ## Setting up the environment 
```

Comment:
```suggestion
## Set up the environment 
```

Reply:
In the guideline, it has been suggested to use the `-ing` form in the titles.

Reply:
The guideline has been revised to use simple present tense